### PR TITLE
Create orders table #11

### PR DESCRIPTION
**Pull request by knoebelja**

### CODE REVIEW

**Comment by onggunhao** [comment]
closes #11

File: server/db/models/order.spec.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  })
+
+  describe('definitions', () => {
+
+    it('has expected quantity definition', () => {
+      expect(Order.attributes.quantity).to.be.an('object')
+    });
+
+    it('has expected price definition', () => {
+      expect(Order.attributes.quantity).to.be.an('object')
```
Comment (emshuttles) [inline server/db/models/order.spec.js]: Should say price instead of quantity

File: server/db/models/order.spec.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  })
+
+  describe('Getter Methods', () => {
+
+    const order1 = Order.build({quantity: 5, price: 999})
+
+    describe('total price', () => {
+
+      it('returns the price and total product divided by 100', () => {
+        expect(order1.totalPrice)
```
Comment (emshuttles) [inline server/db/models/order.spec.js]: I think you want to have something like expect(blah).to.equal(blah) here

**Review by emshuttles — COMMENTED** [review]
Just a couple things to look at.

**Review by emshuttles — APPROVED** [review]

**Pull request merged**